soh/src/code/game.c

```diff
@@ -245,10 +245,8 @@ int fbTest = -1;
 void GameState_Update(GameState* gameState) {
     GraphicsContext* gfxCtx = gameState->gfxCtx;
 
-    if (fbTest == -1)
-    {
-        fbTest = gfx_create_framebuffer(64, 112, SCREEN_WIDTH, SCREEN_HEIGHT, true);
-        //fbTest = gfx_create_framebuffer(256, 512);
+    if (fbTest == -1) {
+        fbTest = gfx_create_framebuffer(64, 112, 64, 112, true);
```

Comment:
This makes things even worse then they were before the change - all four parameters are magic values and last bool should be really made enum struct.
Please use some meaningful constants instead of raw magic numbers.

Reply:
Thanks for bringing attention to this. These values have since been defined upstream on decomp, but there is much to update to bring us in line with that and is out of scope of this bug fix PR.

I can look to at least define these two values as temporary consts, but it is worth mentioning that I intend to bring us more in line with some of the kaleido decomp updates in a follow up PR, as well as aligning SoH's kaleido framebuffer handling with the improvements from 2Ship.

Reply:
I brought over the decomp enums for now for width/height.